frontend/app/routes/_gcweb-app.personal-information.phone-number.edit.tsx

```diff
@@ -35,7 +36,9 @@ export async function loader({ request }: LoaderFunctionArgs) {
 
 export async function action({ request }: ActionFunctionArgs) {
   const formDataSchema = z.object({
-    phoneNumber: z.string().refine((val) => isValidPhoneNumber(val, 'CA'), { message: 'Invalid phone number' }),
+    phoneNumber: z.string()
+    .min(1, { message: 'empty-phone-number' })
+    .refine((val) => isValidPhoneNumber(val, 'CA'), { message: "invalid-phone-format" }),
```

Comment:
**Nitpick:** If you're using VSCode, turn on the "Format on Save" option to have consistent formatting when saving a file.